coordinator/src/main/scala/filodb.coordinator/queryplanner/SingleClusterPlanner.scala

```diff
@@ -92,6 +74,65 @@ class SingleClusterPlanner(val dataset: Dataset,
    qContext.plannerParams.targetSchemaProviderOverride.getOrElse(_targetSchemaProvider)
   }
 
+  /**
+   * Returns true iff a target-schema:
+   *   (1) matches any shard-key matched by the argument filters, and
+   *   (2) changes between the argument timestamps.
+   */
+  private def isTargetSchemaChanging(shardKeyFilters: Seq[ColumnFilter],
```

Comment:
A unit test for this method if possible its no longer a one line it was before. Same for ``useTargetSchemaForShards``

Reply:
Added tests for both ✅